baram/ec2_manager.py

```diff
@@ -1,60 +1,252 @@
 import boto3
+import botocore.exceptions
+
+from baram.log_manager import LogManager
 
 
 class EC2Manager(object):
     def __init__(self):
         self.cli = boto3.client('ec2')
+        self.logger = LogManager.get_logger()
 
     def list_security_groups(self):
-        '''
+        """
         Describes the specified security groups or all of your security groups.
 
         :return: SecurityGroups
-        '''
-        return self.cli.describe_security_groups()['SecurityGroups']
+        """
+        security_groups = self.cli.describe_security_groups()['SecurityGroups']
+        result = [x['GroupId'] for x in security_groups]
+        return set(result)
+
+    def list_instances(self):
+        """
+
+
+        :return:
+        """
```

Comment:
1. unnecessary new lines 
2. please describe the method as comment.

Reply:
The method `describe_instance` in this class could cover this method, so this will be deleted in next commit. Thanks.